beta-pipelines/skylab/m3c/Dockers/m3C-yap-hisat-docker/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM mambaorg/micromamba:0.23.0
```

Comment:
Just want to confirm - this is the same Dockerfile that was used to build the latest Docker used in the WDL? I know we played with a lot of changes to directory structure, etc., so just double checking to be sure.